src/main/java/com/puppycrawl/tools/checkstyle/AstTreeStringPrinter.java

```diff
@@ -41,6 +41,20 @@
  */
 public final class AstTreeStringPrinter {
 
+    /**
+     * Enum to be used for test if comments should be printed.
+     */
+    public enum PrintOptions {
+        /**
+         * Comments has to be printed.
+         */
+        COMMENTS,
+        /**
+         * Comments has NOT be printed.
+         */
+        NONE
+    }
```

Comment:
> printFileAst(file, PrintOptions.NONE);

this line tell user nothing clear in what it is intended to ..... 
`print File Ast OF file with none print options`.
it should be written as 
`print File Ast Of file without comments print options`
so java code is `printFileAst(file, PrintOptions.WITHOUT_COMMENTS)`

So values should be:
```
PrintOptions.WITHOUT_COMMENTS
PrintOptions.WITH_COMMENTS
```

@rnveach , please confirm.